app/utils/webhooks.py

```diff
@@ -1,11 +1,38 @@
+import re
+from collections.abc import Callable
 from io import BytesIO
 
 import discord
 
+from app.setup import bot
 from app.utils.message_data import MessageData, scrape_message_data
 
 GuildTextChannel = discord.TextChannel | discord.Thread
 
+_EMOJI_REGEX = re.compile(r"<(a?):(\w+):(\d+)>", re.ASCII)
+
+
+def _convert_nitro_emojis(s: str, *, force: bool = False) -> str:
+    """
+    Converts a custom emoji to a concealed hyperlink.  Set `force` to True
+    to convert emojis in the current guild too.
+    """
+    guild = next(g for g in bot.guilds if "ghostty" in g.name.casefold())
```

Comment:
Maybe this could be its own util too since it's the same as `entity_mentions/fmt.py:33`? `get_guild()` or whatever